proto/event.proto

```diff
@@ -157,7 +163,7 @@ message TransactionStatusMeta {
   uint64 fee = 3;
   repeated uint64 pre_balances = 4;
   repeated uint64 post_balances = 5;
-  repeated InnerInstruction inner_instructions = 6;
+  repeated InnerInstructions inner_instructions = 6;
```

Comment:
Do we still need `repeated` here?

Reply:
It seems to be a vec, but will take another look

Reply:
https://github.com/solana-labs/solana/blob/master/ledger/src/blockstore.rs#LL7101C21-L7101C21

```rust
        let inner_instructions_vec = vec![InnerInstructions {
            index: 0,
            instructions: vec![InnerInstruction {
                instruction: CompiledInstruction::new(1, &(), vec![0]),
                stack_height: Some(2),
            }],
        }];
```
Looks like two nested vecs